packages/manager/apps/web-hosting/src/utils/validator.ts

```diff
@@ -0,0 +1,72 @@
+import punycode from 'punycode';
+import ipaddr from 'ipaddr.js';
+
+export const isValidIpv4 = function isValidIpv4(ip: string) {
+  if (ip != null && ip.split('.').length === 4) {
```

Comment:
do you need to do these checks since you have the `ipaddr.IPv4.isValid` method to check the validity ?